api/src/test/java/org/pmiops/workbench/access/AccessTierServiceTest.java

```diff
@@ -329,4 +354,12 @@ public void test_addUserToAllTiers_two() {
     assertThat(controlledMembership.getAccessTier()).isEqualTo(controlledTier);
     assertThat(controlledMembership.getTierAccessStatusEnum()).isEqualTo(TierAccessStatus.ENABLED);
   }
+
+  private void sleepMillis(long milliseconds) {
```

Comment:
Should really avoid sleeping in tests unless there are no other options. In this case, I would probably either remove the time component from the model class, or configure a way in which you can stub out a clock for testing.

Changing usages to something like `setLastUpdated(clock.now())` then injecting a fake clock into the service, would be one way

Reply:
right.  Updated to use FakeClock.